api/services/control/control.proto

```diff
@@ -227,11 +228,17 @@ message Descriptor {
 }
 
 message BuildResultInfo {
-	Descriptor Result = 1;
+	Descriptor ResultDeprecated = 1;
```

Comment:
Like we do in https://github.com/moby/buildkit/blob/cab7db27074d275af1eeeaf12c3bd7bc8266cb88/api/services/control/control.proto#L202-L203

Can we keep `Result`? For a single response it would fill both `Result` and `Results` but for multiple responses only `Results` to keep backward compat?

Reply:
At the moment it always fills both, putting `Results[0]` into `ResultDeprecated`. The client-side logic is the same either way, no strong opinions either way:

- If `len(Results) > 0`, `for _, result := range Results`
- Else `result = ResultDeprecated`